docs/web-app-samples.md

```diff
@@ -110,20 +106,11 @@ The following configuration is applicable for below scenario:
                   <!-- Java Runtime Stack for Web App on Linux-->
                   <linuxRuntime>tomcat 8.5-jre8</linuxRuntime>
                   
-                  <!-- FTP deployment -->
-                  <deploymentType>ftp</deploymentType>
-                  <!-- Resources to be deployed to your Web App -->
-                  <resources>
-                     <resource>
-                        <!-- Where your artifacts are stored -->
-                        <directory>${project.basedir}/target</directory>
-                        <!-- Relative path to /site/wwwroot/ -->
-                        <targetPath>webapps</targetPath>
-                        <includes>
-                           <include>*.war</include>
-                        </includes>
-                     </resource>
-                  </resources>
+                  <!-- WAR deployment -->
+                  <deploymentType>war</deploymentType>
+
+                  <!-- Specify the war file location, optional if the war file location is: ${project.build.directory}/${project.build.finalName}.war -->
```

Comment:
How about  "If it is not specified, ${project.build.directory}/${project.build.finalName}.war will be used as default."?

Reply:
Yeah, that's much better